### PR TITLE
Add roadmap to documentation

### PR DESCRIPTION
**Pull request by woop**
Adds our current roadmap to Feast our documentation

```release-note
NONE
```

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
 # Roadmap
 
+Our current proposed roadmap for 0.11 and onward is as follows
+
+## Backlog
+
+- Add On demand transformations support
+- Add Data quality monitoring
+- Add Snowflake offline store support
+- Add Bigtable support
```
Comment (tedhtchang) [inline docs/roadmap.md:10]: Is this for online store support?

Comment (jklegar) [reply docs/roadmap.md:10]: yep!

**Review by jklegar — APPROVED** [review]

**Pull request merged**